Sources/LanguageServerProtocol/CodeActionKind.swift

```diff
@@ -22,25 +22,25 @@ public struct CodeActionKind: RawRepresentable, Codable, Hashable {
   }
 
   /// QuickFix action, such as FixIt.
-  public static let quickFix: CodeActionKind = CodeActionKind(rawValue: "quickfix")
```

Comment:
Removal of type annotations on public properties has been suggested before, and the consensus seems to be to keep them explicit: https://github.com/apple/sourcekit-lsp/pull/3#issuecomment-438743769

Reply:
Fair enough. I'll go ahead and just remove it on internal properties then.